src/app/components/community/sidebar/sidebar.ts

```diff
@@ -103,6 +108,26 @@ export default class AppCommunitySidebar extends Vue {
 		return mods;
 	}
 
+	get shouldShowGames() {
+		return this.community.games && this.community.games.length;
+	}
+
+	get hasMoreGames() {
+		return this.community.games && this.community.games.length > GAME_LIST_COLLAPSED_COUNT;
+	}
+
+	get visibleGames() {
+		if (!this.community.games) {
+			return [];
+		}
+
+		if (this.gameListCollapsed) {
+			return this.community.games.slice(0, GAME_LIST_COLLAPSED_COUNT);
+		}
+
+		return this.community.games;
+	}
```

Comment:
If we go with the suggestion to change `community.games` this function name would be ambiguous between visible in the list or non-hidden games. Rename or add a comment if needed.